### PR TITLE
properly pass `options` to `getDefaultValues`

### PR DESCRIPTION
**Pull request by servel333**
Fixed call in `deprecated_request_wrap` to properly pass `options` to `getDefaultValues`

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @servel333 isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by servel333** [comment]
I thought I signed it. Let me try again.

**Comment by servel333** [comment]
Ok, I think I signed the right thing.

**Comment by murgatroid99** [comment]
OK, that check is green now so it looks like it worked this time.
